Cargo.toml

```diff
@@ -149,7 +149,7 @@ edition = "2021"
 homepage = "https://lurk-lang.org/"
 license = "MIT OR Apache-2.0"
 repository = "https://github.com/lurk-lab/lurk-rs"
-rust-version = "1.72"
+rust-version = "1.74.1"
```

Comment:
Note: if you make the lurk package align with the workspace version, you'll only have to edit this in one place next time this happens.

Reply:
Well, the merge is still failing so... should I revert the change in the other line of this file?

Reply:
Not 'revert the change', rather make it 'rust-version.workspace = true'

Reply:
@huitseeker we unfortunately can't use the workspace version yet due to a `cargo msrv` limitation. There is an open PR though: https://github.com/foresterre/cargo-msrv/issues/295

Reply:
Factored out to #1110